lookout/style/format/rules.py

```diff
@@ -134,13 +140,17 @@ def predict(self, X: numpy.ndarray, vnodes_y: Sequence[VirtualNode],
             postprocess = import_module(
                 "lookout.style.format.langs.%s.postprocessor" % feature_extractor.language) \
                 .postprocess
+            filter_uast_breaking_preds = import_module(
```

Comment:
we can just import normally here since we use the same function for all languages.